manifests/config.pp

```diff
@@ -9,6 +9,10 @@
     groups => $candlepin::user_groups,
```

Comment:
Should we also manage the primary group on the user while we're at it? This creates the correct autorequire relationship `Group[$group] -> User[$user]`:
```suggestion
    gid    => $candlepin::group,
    groups => $candlepin::user_groups,
```